packages/fscommerce/src/Commerce/types/Cart.ts

```diff
@@ -37,6 +37,12 @@ export interface Cart<T extends CartItem = CartItem> {
    */
   items: T[];
 
+  /**
+   * An array of products with detailed info for variants in the cart, their master
+   * products and bonus products in the cart.
+   */
+  product_details?: Product[];
```

Comment:
All of our other property names are camelCased, so this should be too.

Reply:
@nathan-sankbeil updated